packages/grid/_modules_/grid/utils/utils.ts

```diff
@@ -15,23 +15,6 @@ export function isObject(value: any): value is Record<string, any> {
   return typeof value === 'object';
 }
 
-// TODO v5: remove
-export function getThemePaletteMode(palette: any): string {
-  return palette.type || palette.mode;
-}
-
-export function getMuiVersion(): string {
-  if (!('fade' in styles)) {
-    return 'v5';
-  }
-
-  if ('fade' in styles && 'alpha' in styles) {
-    return 'v4.12';
-  }
-
-  return 'v4';
-}
-
 export function muiStyleAlpha(color: string, value: number): string {
```

Comment:
This helper can be removed now.

Reply:
Yes some changes needed you migration to `@mui/`, I'll complet this PR :+1: